tests/UnitTests/BatchedDogStatsdTest.php

```diff
@@ -74,4 +74,34 @@ public function testReportSendsOnceBufferIsFilled()
             array("metrics" => 3)
         );
     }
+
+    public function testGetBufferLength() {
+        // Given
```

Comment:
NIT: I'd prefer we remove these comments (on each line, for both tests) as it doesn't fit in with the rest of the code. 
Otherwise, this change LGTM.

Reply:
thanks! Removed.